tests/constants/githubissue.constants.ts

```diff
@@ -187,3 +187,69 @@ export const ISSUE_PENDING_MODERATION = new GithubIssue({
   },
   comments: [PENDING_TUTOR_MODERATION]
 });
+
+export const ISSUE_PENDING_MODERATION_HIGH_SEVERITY_FEATURE_FLAW = new GithubIssue({
+  id: '239538360',
+  number: 93,
+  assignees: [
+    {
+      login: 'geshuming',
+      id: 43642522,
+      url: 'https://api.github.com/users/geshuming'
+    }
+  ],
+  body:
+    '# Issue Description\n{original issue description}\n' +
+    "# Team's Response\n{team's response}\n # Disputes\n\n" +
+    "## :question: Issue Type\n\n### Team says:\r\n{the team's action that is being disputed}\r\n\r\n" +
+    "### Tester says:\r\n{tester's objection}\n\n-------------------\n## :question: Issue Severity\n\n" +
+    "### Team says:\r\n{the team's action that is being disputed}\r\n\r\n" +
+    "### Tester says:\r\n{tester's objection}\n\n-------------------\n## :question: Not Related Question\n\n" +
+    "### Team says:\r\n{the team's action that is being disputed}\r\n\r\n" +
+    "### Tester says:\r\n{tester's objection}\n\n-------------------\n\n",
+  created_at: '2020-10-14T10:28:32Z',
+  labels: [GITHUB_LABEL_TEAM_LABEL, GITHUB_LABEL_TUTORIAL_LABEL, GITHUB_LABEL_FEATURE_FLAW, GITHUB_LABEL_HIGH_SEVERITY],
+  state: IssueState.Open,
+  title: 'Redundant buggy feature',
+  updated_at: '2020-11-25T13:19:40Z',
+  url: 'https://api.github.com/repos/CATcher-org/pe-results/issues/26',
+  user: {
+    login: 'geshuming',
+    avatar_url: 'https://avatars0.githubusercontent.com/u/43642522?v=4',
+    url: 'https://api.github.com/users/geshuming'
+  },
+  comments: [PENDING_TUTOR_MODERATION]
+});
```

Comment:
Small nit on my side, you can abstract the input parameter for `new GithubIssue` into another variable and use destructuring to reduce copying

Reply:
I agree that that would be a good way to reduce copying, but not sure if that would make it easier for viewing! Since this is in `githubissue.constants.ts` it might be better to just display the contents fully, so we don't have to refer back to the variable to check the contents. What do the others think?

Reply:
In my honest opinion, given the body and other fields are the same, destructuring is preferred here. Don't think readability will be a big issue there 😄

Reply:
Alright thanks for your suggestions!